pytorch/inference/docker/1.9/py3/Dockerfile.graviton

```diff
@@ -1,4 +1,139 @@
-FROM ubuntu:18.04
+FROM arm64v8/ubuntu:20.04
 
-# Placeholder Dockerfile for Graviton
-RUN apt-get update
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
No changed needed here, but making a note on this PR for future reference:

Ensure that MME is possible with Graviton endpoints on SageMaker when the feature is ready. If MME is not available for Graviton endpoints, remove the `capabilities.multi-models` LABEL from the dockerfile.